guides/release/testing/testing-components.md

```diff
@@ -317,17 +317,18 @@ import { hbs } from 'ember-cli-htmlbars';
 import Service from '@ember/service';
 
 //Stub location service
-const locationStub = Service.extend({
-  city: 'New York',
-  country: 'USA',
-  currentLocation: {
+class LocationStub extends Service {
+  city = 'New York';
+  country = 'USA';
+  currentLocation = {
     x: 1234,
     y: 5678
-  },
+  };
 
   getCurrentCity() {
     return this.city;
-  },
+  }
+
   getCurrentCountry() {
     return this.country;
   }
```

Comment:
In the line after this one, it should be just `}`. You have a stray `)` and the linter doesn't like the semicolon. Sorry I can't comment on that line directly - it's not part of the diff.

Reply:
thanks!